src/agent/main.cpp

```diff
@@ -47,7 +50,13 @@
 #include "common/code_utils.hpp"
 #include "common/logging.hpp"
 #include "common/types.hpp"
+#include "dbus/dbus_agent.hpp"
+
+using namespace ot::BorderRouter::Ncp;
+using namespace otbr::agent;
+using namespace otbr::dbus;
```

Comment:
[Do not use using-directives](https://google.github.io/styleguide/cppguide.html#Namespaces).